llvm/test/Transforms/SROA/vector-promotion.ll

```diff
@@ -1227,6 +1227,166 @@ define void @swap-15bytes(ptr %x, ptr %y) {
   ret void
 }
 
+define <4 x i32> @ptrLoadStoreTys(ptr %init, i32 %val2) {
+; CHECK-LABEL: @ptrLoadStoreTys(
```

Comment:
Looks like the full test is now squashed? I expected to only see smaller diffs

Reply:
Yes it is squashed -- I will undo the squash.

Reply:
done

Reply:
I meant pre-push the add baseline tests as its own standalone commit directly to main. With the project setting of squash and submit, there's less point in splitting out the commits for submit